ppanggolin/annotate/annotate.py

```diff
@@ -392,24 +401,30 @@ def annotatePangenome(pangenome, fastaList, tmpdir, cpu, translation_table="11",
 def launch(args):
     filename = mkFilename(args.basename, args.output, args.force)
     pangenome = Pangenome()
-    if args.fasta is not None and args.anno is None:
-        annotatePangenome(pangenome, args.fasta, args.tmpdir, args.cpu,  args.translation_table, args.kingdom, args.norna, args.overlap)
+    if args.fasta_list is not None and args.anno is None:
+        pangenome = Pangenome()
+        annotatePangenome(pangenome, args.fasta_list, args.tmpdir, args.cpu, args.translation_table, args.kingdom, args.norna, args.overlap, fastaList=True)
+    elif args.fasta_files is not None and args.anno is None:
+        pangenome = Pangenome()
+        annotatePangenome(pangenome, args.fasta_files, args.tmpdir, args.cpu, args.translation_table, args.kingdom, args.norna, args.overlap, fastaList=False)
     elif args.anno is not None:
         readAnnotations(pangenome, args.anno, pseudo = args.use_pseudo)
         if pangenome.status["geneSequences"] == "No":
-            if args.fasta:
-                getGeneSequencesFromFastas(pangenome, args.fasta)
-            else:
+            if args.fasta_list is None and args.fasta_files is None:
                 logging.getLogger().warning("You provided gff files without sequences, and you did not provide fasta sequences. Thus it was not possible to get the gene sequences.")
                 logging.getLogger().warning("You will be able to proceed with your analysis ONLY if you provide the clustering results in the next step.")
-
+            elif pangenome.status["geneSequences"] == "No" and args.fasta_list is not None:
+                getGeneSequencesFromFastas(pangenome, args.fasta_list)
+            elif pangenome.status["geneSequences"] == "No" and args.fasta_files is not None:
+                getGeneSequencesFromFastas(pangenome, args.fasta_files)
```

Comment:
This is akin to the comment I made a few minutes ago. This is for the case where we provide annotations through the command line and fasta files through --fasta_files.
This assumes that the organism names are the same in both provided ways, so that the names in the annotation list file are fasta filepaths, which feels weird.
I suggest to allow the use of --fasta_files only on its own ? The combinations with other inputs will be unpractical otherwise I think

Reply:
I realize that the fact that organism names need to be identical is really unclear when reading the code, I'll try to improve the comments on this part